tests/fixtures/parsers/type-annotations/variable-declaration-init-type-annotation-no-space.js

```diff
@@ -0,0 +1,294 @@
+/**
+ * Parser: typescript-eslint-parser v2.0.0 (TS 2.2.1)
+ * Source code:
+ * var a: Foo= b;
+ */
+
+exports.parse = () => ({
+    "type": "Program",
+    "range": [
+        0,
+        14
+    ],
+    "loc": {
+        "start": {
+            "line": 1,
+            "column": 0
+        },
+        "end": {
+            "line": 1,
+            "column": 14
+        }
+    },
+    "body": [
+        {
+            "type": "VariableDeclaration",
+            "range": [
+                0,
+                14
+            ],
+            "loc": {
+                "start": {
+                    "line": 1,
+                    "column": 0
+                },
+                "end": {
+                    "line": 1,
+                    "column": 14
+                }
+            },
+            "declarations": [
+                {
+                    "type": "VariableDeclarator",
+                    "range": [
+                        4,
+                        13
+                    ],
+                    "loc": {
+                        "start": {
+                            "line": 1,
+                            "column": 4
+                        },
+                        "end": {
+                            "line": 1,
+                            "column": 13
+                        }
+                    },
+                    "id": {
+                        "type": "Identifier",
+                        "range": [
+                            4,
+                            5
+                        ],
+                        "loc": {
+                            "start": {
+                                "line": 1,
+                                "column": 4
+                            },
+                            "end": {
+                                "line": 1,
+                                "column": 5
+                            }
+                        },
+                        "name": "a",
+                        "typeAnnotation": {
+                            "type": "TypeAnnotation",
+                            "loc": {
+                                "start": {
+                                    "line": 1,
+                                    "column": 7
+                                },
+                                "end": {
+                                    "line": 1,
+                                    "column": 10
+                                }
+                            },
+                            "range": [
+                                7,
+                                10
+                            ],
+                            "typeAnnotation": {
+                                "type": "TSTypeReference",
```

Comment:
It seems like this file still has TypeScript-specific properties in the AST, while the others don't. Is this intentional?